language/de/ucp.php

```diff
@@ -447,20 +447,22 @@
 	'NO_OLDER_PM'				=> 'Keine älteren Nachrichten.',
 	'NO_PASSWORD_SUPPLIED'		=> 'Du kannst dich nicht ohne Passwort anmelden.',
 	'NO_RECIPIENT'				=> 'Kein Empfänger angegeben.',
+	'NO_RESET_TOKEN'			=> 'Kein Token zum Zurücksetzen des Passworts angegeben.',
 	'NO_RULES_DEFINED'			=> 'Keine Regeln festgelegt.',
 	'NO_SAVED_DRAFTS'			=> 'Keine Entwürfe gespeichert.',
 	'NO_TO_RECIPIENT'			=> 'Keinen',
 	'NO_WATCHED_FORUMS'			=> 'Du hast keine Foren abonniert.',
 	'NO_WATCHED_SELECTED'		=> 'Du hast keine abonnierten Themen oder Foren ausgewählt.',
 	'NO_WATCHED_TOPICS'			=> 'Du hast keine Themen abonniert.',
 
-	'PASS_TYPE_ALPHA_EXPLAIN'	=> 'Das Passwort muss zwischen %1$s und %2$s lang sein und aus Groß- und Kleinbuchstaben sowie Ziffern bestehen.',
-	'PASS_TYPE_ANY_EXPLAIN'		=> 'Muss zwischen %1$s und %2$s lang sein.',
-	'PASS_TYPE_CASE_EXPLAIN'	=> 'Das Passwort muss zwischen %1$s und %2$s lang sein und muss aus Groß- und Kleinbuchstaben bestehen.',
-	'PASS_TYPE_SYMBOL_EXPLAIN'	=> 'Das Passwort muss zwischen %1$s und %2$s lang sein und muss aus Groß- und Kleinbuchstaben, Ziffern sowie Sonderzeichen bestehen.',
+	'PASS_TYPE_ALPHA_EXPLAIN'	=> 'Das Passwort muss mindestens %1$s Zeichen lang sein und aus Groß- und Kleinbuchstaben sowie Ziffern bestehen.',
```

Comment:
Muss hier nicht das "Zeichen" weg? Das ist im englischen auch nicht vorhanden und wird vermutlich an anderer Stelle generiert. Der Platzhalter ist hier ja auch ein Text und kein String

Reply:
Dann haben wir:

> Das Passwort muss mindestens 6 lang sein und aus Groß- und Kleinbuchstaben sowie Ziffern bestehen.

Klingt doch unvollständig.

Reply:
Das ist aber im Englischen nicht besser:
> Password must be between %1$s and %2$s long and must contain letters in mixed case.
Ich hab da eher das Gefühl, dass das in zwei Schritten erfolgt: erst wird der Text "6 Zeichen" gebildet und dieser dann in diesen Text eingefügt.

Reply:
Ich fürchte, du hast vermutlich recht...